README.md

```diff
@@ -21,77 +21,44 @@ After responding to changes, merge in GitHub.
 
 ## Setup
 
-### API
-
-* Install JDK 8 on your machine from http://www.oracle.com/technetwork/java/javase/downloads/index.html.
-* Set your JAVA_HOME environment variable to point at the installation directory.
+Docker must be installed to build and run code.
 
 ### UI
 
 * Direct your editor to write swap files outside the source tree, so Webpack
 does not reload when they're updated.
 [Example for vim](https://github.com/angular/angular-cli/issues/4593).
-* Install npm. (For Google workstations, see go/nodejs.)
-
-### General
-
-* Install Google Cloud SDK on your device from https://cloud.google.com/sdk/downloads.
-* If not selected initially, add the AppEngine component with `gcloud components install app-engine-java`.
-* Run tools/setup_env.sh. (This installs angular-cli globally. You may also get it from `ui/node_modules` after
-running `npm install`.)
 
 ## Running the Dev Servers
 
 ### API: dev AppEngine appserver
 
 From the `api/` directory:
-
 ```Shell
-./gradlew appengineRun
+./project.rb dev_up
 ```
 
 When the console displays "Dev App Server is now running", you can hit your
 local API server under http://localhost:8081/api/.
 
 ### UI
 
-From the `ui/` subdirectory:
-
+From the `ui/` directory:
 ```Shell
-npm run start [-- --environment test]
+./project.rb dev_up
 ```
 
 After webpack finishes the build, you can view your local UI server at
-http://localhost:4200/.
+http://localhost:4200/. You can view the tests at http://localhost:9876/debug.html.
 
-By default, this connects to a local API server. Use `--environment=$ENV` to
+By default, this connects to a test API server. Use `--environment=$ENV` to
```

Comment:
So if you're making UI only changes, this seems fine.

If you're making API changes and want to make sure that the UI still works as-is, or if you're making both UI and API changes in the same PR, it isn't -- we need to run a local API server.

We should definitely make both simple; I just kind of liked having connection to your local server be the default, since it should always work (whether or not you're making API changes.)